en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+* Perform an initial assessment of the report.
+* [Create a issue in our private security repository](https://github.com/silverstripe-security/security-issues/issues/new) unless to report is obviously invalid. e.g. SPAM.
+* If encrypted information is provided, attach it to the private security issue.
+* Reply to [security@silverstripe.org](mailto:security@silverstripe.org) ONLY and provide a link to the private security issue. Keep most of the discussion on GitHub.
+* Perform initial criticality assessment. Validate assessment with at least one other member of the security team before replying to the reporter with your conclusion. Ensure the reporter is given a justification for all issues we classify or demote as non-security vulnerabilities. You may need to seek additional information from the reporter before completing the criticality assessment.
+* Add the new issue in the "Backlog" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Use the [CVSS Calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) to determine the issue severity.
+* Once the issue is confirmed, create a _draft security advisory_ against the relevant GitHub repository. This will allow you to request a CVE.
+* Once a CVE has been assigned, respond to issue reporter and add it to the Github issue.
+* Ask the reporter if they want to be credited for the disclosure and under what name.
+* Clarify who picks up and owns the issue (assign in Github).
+  The owner can be separate from the developer resolving the issue,
+  their primary responsibility is to ensure the issue keeps moving through the process correctly.
+
+### When developing a fix
+
+* Ensure you're working on the oldest supported minor release branch of every supported major release (see [Supported Versions](#supported-versions))
+* Move the issue into "In Progress" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+* Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo. Don't forget to update branches from the upstream repo.
+* Ensure that all security commit messages are prefixed with the CVE. E.g. "[CVE-2019-001] Fixed invalid XSS"
+
+### Before release
+
+* For issues rated "high" or "critical" (CVSS of >=7.0), post a pre-announcement to the [security pre-announcement list](https://groups.google.com/a/silverstripe.com/forum/#!forum/security-preannounce).
+  It should include a basic "preannouncement description" which doesn't give away too much,
+  the CVSS score as well as the CVE identifier.
+* Create a draft page under [Open Source > Download > Security Releases](https://www.silverstripe.org/admin/pages/edit/show/794).
+  Populate it with the information from the [Github project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Link to silverstripe.org security release page in the changelog.
```

Comment:
```suggestion
* Link to the silverstripe.org security release page in the changelog.
```
Do you mean https://www.silverstripe.org/download/security-releases/ or do you mean the specific page for the individual vulnerability (e.g. https://www.silverstripe.org/download/security-releases/cve-2022-28803)?